.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -76,19 +77,24 @@ Zebra's Rust API doesn't have any support or stability guarantees, so we keep al
 
 <details>
 
-<summary>If you're publishing crates for the first time:</summary>
+<summary>If you're publishing crates for the first time, click this triangle for extra steps</summary>
 
 - [ ] Install `cargo-release`: `cargo install cargo-release`
 - [ ] Make sure you are  an owner of the crate or [a member of the Zebra crates.io `owners` group on GitHub](https://github.com/orgs/ZcashFoundation/teams/owners)
 
 </details>
 
-- [ ] Update crate versions and do a release dry-run
-    - [ ] `cargo clean` (optional)
-    - [ ] `cargo release version --verbose --execute --workspace --exclude zebrad beta`
-    - [ ] `cargo release version --verbose --execute --package zebrad [ major | minor | patch ]`
-    - [ ] `cargo release publish --verbose --dry-run --workspace`
-- [ ] Commit the version changes to your release PR branch using `git`: `cargo release commit --verbose --execute --workspace`
+Check that the release will work:
+- [ ] Update crate versions, commit the changes to the release branch, and do a release dry-run:
+
+```sh
+cargo clean # optional
+cargo release version --verbose --execute --workspace --exclude zebrad beta
+cargo release version --verbose --execute --package zebrad patch # [ major | minor | patch ]
+cargo release replace --verbose --execute --package zebrad
+cargo release commit --verbose --execute
+cargo release publish --verbose --dry-run --workspace
```

Comment:
I don't think step will work for unpublished crates that depend on other unpublished crates.

```suggestion
```

Reply:
I just did the release with this step and it works.

The incorrect part of the script is actually the version bumps.

Reply:
It works, but some crates encounter a bug in `cargo release`, so I skipped them:
https://github.com/crate-ci/cargo-release/issues/691

Reply:
We should move the tests that use zebra-consensus to avoid the circular dependency later.

Reply:
This unresolved conversation is blocking the merge... Should it be resolved?

Reply:
> We should move the tests that use zebra-consensus to avoid the circular dependency later.

I tried removing the dev-dependency entirely, and the bug still happens. So unfortunately I don't think moving the tests would fix it!

(But making a copy of the zebra-consensus verifiers in tower-batch-control is a good idea for crater runs, I'll add that to ticket #6924.)